src/components/signal-viewer/renderer.tsx

```diff
@@ -18,7 +20,7 @@ export default class SignalViewer extends React.Component<Props> {
   public render() {
     return (
       <div className="signal-viewer">
-        <div className="signal-table">
+        <div className="data-table">
```

Comment:
This is not acceptable. Let's find a new name that encompasses the different tables.

Reply:
@domoritz I have used the same names since I need to apply the same styles. If I change it, merging styles wouldn't be possible.

Reply:
We should have a separate type for `tables`.

Reply:
I am thinking to name them `debug-table`.

Reply:
Why not just create a style for `table` and not even add a class?